Constitution.txt

```diff
@@ -1,490 +1,264 @@
+3: Every other Bill which has been presented to the House of Representatives and the Senate, and has been approved and signed by all Representatives and Senators from several of the States, shall be sent to the Chief Executive of each of those States. If he should approve the Bill, it shall be considered approved by his State; if not, he shall send it to the Legislature of his State. If, after consideration, the Legislature of the State approves and signs the Bill in keeping with that State's Constitution, it shall be considered approved by the State. When at least two States have approved the Bill, it shall become law in those states which have so approved it, for a period not to exceed six years from first approval.
 
 Section 8
-1: The Congress shall have Power To lay and collect Taxes, Duties, Imposts and Excises, to pay the Debts and provide for the common Defence and general Welfare of the United States; but all Duties, Imposts and Excises shall be uniform throughout the United States;
+The Congress shall have Power:
 
-2: To borrow Money on the credit of the United States;
+1:  To lay and collect Taxes, Duties, Imposts, and Excises, on commerce among the several States and with foreign nations; to pay the Debts and provide for the common Defense of these United States; but all Duties, Imposts and Excises shall be uniform throughout these United States; and no Tax, Duty, Impost, or Excise shall apply to transactions occurring wholly within one State, nor shall any resident of any State be compelled by law to participate in a transaction other than the payment of such Taxes, Duties, Imposts, and Excises.
 
-3: To regulate Commerce with foreign Nations, and among the several States, and with the Indian Tribes;
+2: To borrow Money on the credit of these United States, when such credit shall be approved and a payment plan funded by laws passed as described in Article I, Section 7, Part 2;
 
-4: To establish an uniform Rule of Naturalization, and uniform Laws on the subject of Bankruptcies throughout the United States;
+3: To regulate Commerce with foreign Nations, and that which crosses borders among the several States;
 
-5: To coin Money, regulate the Value thereof, and of foreign Coin, and fix the Standard of Weights and Measures;
+4: To establish a uniform Rule of Naturalization, and uniform Laws on the subject of Bankruptcies throughout these United States;
 
-6: To provide for the Punishment of counterfeiting the Securities and current Coin of the United States;
+5: To constitute Tribunals inferior to the supreme Court;
```

Comment:
Note here that original elements 5, 6, 7, and 8 are removed, jumping straight from Naturalization and Bankruptcies to the constitution of Tribunals under the Supreme Court.

I remove the authority of Congress to coin Money. As we learn from the Austrian School of Economics, Money is not the result of government action, but instead is a result of market dynamics identifying goods which are widely considered valuable, fungible, and divisible. Similarly, Weights and Measures are determined in the course of human communication, as those measures which are broadly useful. Congress may make reference to commonly-understood weights and measures, and define in its Bills the amounts of money to be spent or taxed based on those commonly-understood weights and measures, with the money being those goods which are considered money by ordinary people.

Since the government has no authority to coin money, it therefore has no authority to determine a given currency to be Valid or Counterfeit.

Post Offices and Roads may be established by private citizens, or by local governments if their state Constitutional allows it. Congress will have authority over the borders between states, via interstate Commerce, and so can resolve issues arising from localities connecting roads to one another.

Many will consider the removal of element 8 to be extremely controversial. In its removal, I remove the basis for Intellectual Property law. I know that few will accept its removal, and of any change in this constitution, I'm least attached to this one, because while I believe the private-property based argument against IP to be correct and valid, I know I'm in a significant minority.